doc/cascadia/SettingsSchema.md

```diff
@@ -133,3 +133,48 @@ Bindings listed below are per the implementation in `src/cascadia/TerminalApp/Ap
 - moveFocusUp
 - moveFocusDown
 
+## Background Images and Icons
+Some Terminal settings allow you to specify custom background images and icons. It is recommended that custom images and icons are stored in system-provided folders, and are referred to using the correct [URI Schemes](https://docs.microsoft.com/en-us/windows/uwp/app-resources/uri-schemes). URI Schemes provide a way to reference files independent of their physical paths (which may change in the future).
+
+The most useful URI schemes to remember when customizing background images and icons are:
+
+| URI Scheme | Corresponding Physical Path | Use / description |
+| --- | --- | ---|
+| `ms-appdata:///Local/` | `%localappdata%\Packages\Microsoft.WindowsTerminal_8wekyb3d8bbwe\LocalState\` | Per-machine files |
+| `ms-appdata:///Roaming/` | `%localappdata%\Packages\Microsoft.WindowsTerminal_8wekyb3d8bbwe\RoamingState\` | Common files |
+
+> ⚠ Note: Do not rely on file references using the `ms-appx` URI Scheme (i.e. icons). These files are considered an internal implementation detail and may change name/location or may be omitted in the future.
+
+### Icons
+Terminal displays icons for each of your profiles. When it is first run, Terminal generates profiles for any of your built-in shells - PowerShell Core, PowerShell, and any installed Linux/WSL distros, each referring to a stock icon. 
+
+> ⚠ Note: These stock icons are referred to via the `ms-appx` URI Scheme. Do not rely on the files referenced by the `ms-appx` URI Scheme - they are considered an internal implementation detail and may change name/location or may be omitted in the future. 
+
+You can refer to you own icon files if you wish, e.g.:
+
+```json 
+    "icon" : "C:\\Users\\richturn\\OneDrive\\WindowsTerminal\\icon-ubuntu-32.png",
+```
+
+> 👉 Tip: Icons should be sized to 32x32px, in an appropriate raster image format (e.g. .PNG, .GIF, or .ICO) to avoid having to scale your icons during runtime, causing a noticeable delay and loss of quality.

```

Comment:
```suggestion
> 👉 Tip: Icons should be sized to 32x32px in an appropriate raster image format (e.g. .PNG, .GIF, or .ICO) to avoid having to scale your icons during runtime (causing a noticeable delay and loss of quality.)
```